src/bin/nydus-image/main.rs

```diff
@@ -765,6 +796,18 @@ impl Command {
         Ok(())
     }
 
+    fn unpack(args: &clap::ArgMatches) -> Result<()> {
+        let bootstrap = args.value_of("bootstrap").expect("pass in bootstrap");
+        let blob = args.value_of("blob");
+        let output = args.value_of("output").expect("pass in output");
+
+        let unpacker = Box::new(
```

Comment:
let unpacker = OCIUnpacker::new(bootstrap, blob, output).with_context(|| "fail to create unpacker")?;
Seems `Box` is redundant

Reply:
> let unpacker = OCIUnpacker::new(bootstrap, blob, output).with_context(|| "fail to create unpacker")?; Seems `Box` is redundant

Expect unpacker used as trait here. Excluding Box, any other ways?